src/main/cluster-detectors/distribution-detector.ts

```diff
@@ -60,6 +60,14 @@ export class DistributionDetector extends BaseClusterDetector {
       return { value: "custom", accuracy: 10};
     }
 
+    if (this.isCustom()) {
+      return { value: "custom", accuracy: 10};
+    }
+
```

Comment:
I am still confused as to why this PR is adding this? Was it removed in another PR?

Reply:
I'm also confused.. sorry about this (rebase is hard, sometimes). Should be fixed now.